pkg/parsers/kernel/kernel_test_unix.go

```diff
@@ -19,6 +21,7 @@ func assertParseRelease(t *testing.T, release string, b *VersionInfo, result int
 	}
 }
 
+// TestParseRelease tests the ParseRelease() function
```

Comment:
Yeah, it should be unix_test, not test_unix. Renaming. But no harm in golinting as I go ;)